app/controllers/users_controller.rb

```diff
@@ -27,6 +27,11 @@ def demote
     redirect_to url_for(:controller => :users, :action => :index)
   end
 
+  def feedback
+    @posts = Feedback.joins(:post).joins(:feedback_type).select('"posts"."id", "posts"."title", "feedbacks"."created_at", "feedback_types"."short_code", "feedback_types"."color"')
+             .where(:chat_username => @user.username).order(:created_at => :desc).paginate(:page => params[:page], :per_page => 100)
```

Comment:
* Since you have multiple tables, you should qualify any WHERE conditions with their table, rather than leaving them unqualified: `.where(feedbacks: { chat_username: @user.username })`.
 * Is `feedbacks.chat_username` indexed? If not, you should add an index (generate a migration, then use `add_index :feedbacks, :chat_username`) so that this query isn't horribly slow.
 * `.order(created_at: :desc)` will fail here, again because it's unqualified. Specify the table: `.order(Arel.sql('feedbacks.created_at DESC'))`.